ssl/ssl_lib.c

```diff
@@ -5401,6 +5401,34 @@ int SSL_client_hello_get1_extensions_present(SSL *s, int **out, size_t *outlen)
     return 0;
 }
 
+size_t SSL_client_hello_get_raw_extensions(SSL *s, unsigned short *extensions)
```

Comment:
Could you please add a parameter indicating a size of the allocated extensions buffer and return error if the existing extensions do not fit? As the error return should be negative I'd suggest returning an int type.

Reply:
done